interface/HHAnalyzer.h

```diff
@@ -61,25 +61,19 @@ class HHAnalyzer: public Framework::Analyzer {
                 }
             }
         }
+        virtual void endJob(MetadataManager&) override;
 
         // leptons and dileptons stuff
         BRANCH(electrons, std::vector<unsigned int>);
         BRANCH(muons, std::vector<unsigned int>);
         BRANCH(leptons, std::vector<HH::Lepton>);
-        BRANCH(ll, std::vector<HH::Dilepton>);
         BRANCH(met, std::vector<HH::Met>);
-        BRANCH(llmet, std::vector<HH::DileptonMet>);
         BRANCH(jets, std::vector<HH::Jet>);
-        BRANCH(jj, std::vector<HH::Dijet>);
-        BRANCH(llmetjj, std::vector<HH::DileptonMetDijet>);
+        std::vector<HH::Dilepton> ll;
+        std::vector<HH::DileptonMet> llmet;
+        std::vector<HH::Dijet> jj;
+        std::vector<HH::DileptonMetDijet> llmetjj;
```

Comment:
I noticed it was missing for met and llmet so I explicitely added them indeed

the rest is here:
leptons: https://github.com/cp3-llbb/HHAnalysis/blob/2016-01-30.v1/plugins/HHAnalyzer.cc#L77
ll: https://github.com/cp3-llbb/HHAnalysis/blob/2016-01-30.v1/plugins/HHAnalyzer.cc#L78
jets: https://github.com/cp3-llbb/HHAnalysis/blob/2016-01-30.v1/plugins/HHAnalyzer.cc#L420
jj: https://github.com/cp3-llbb/HHAnalysis/blob/2016-01-30.v1/plugins/HHAnalyzer.cc#L508
llmetjj: https://github.com/cp3-llbb/HHAnalysis/blob/2016-01-30.v1/plugins/HHAnalyzer.cc#L634

so except for electrons and muons clearing, everyone should be here now